benefits/core/models/enrollment.py

```diff
@@ -276,10 +281,19 @@ def clean(self):
             # so just create directly for a missing template
             for t in templates:
                 if not template_path(t):
-                    template_errors.append(ValidationError(f"Template not found: {t}"))
+                    errors.append(ValidationError(f"Template not found: {t}"))
+
+            if EnrollmentMethods.IN_PERSON in self.supported_enrollment_methods:
+                try:
+                    in_person_eligibility_context = self.in_person_eligibility_context
+                except KeyError:
+                    in_person_eligibility_context = None
+
+                if not in_person_eligibility_context:
+                    errors.append(ValidationError(f"In-person eligibility context not found for: {self.system_name}"))
```

Comment:
Open to suggestions on this error message :sweat_smile: 

I guess it would help if it told the user some sort of action to take, like to contact a dev or to uncheck `In-person`.

Reply:
Hmmm... yeah, this is a technically accurate error message! But since we show it to end-users, I think we probably want something more friendly here :sweat_smile: 

I agree with your action-oriented take:

> {system_name} not configured for In-person. Please uncheck to continue."

Reply:
I just tested this out and I agree with y'all, adding an action to take and saying that the flow is not configured for In-person, something like Kegan's message, works well for the user. And because the code is explicit that this error occurs when `not in_person_eligibility_context`, it's still easy for a dev to see what exactly is missing in the configuration.

Reply:
Yeah don't worry too much about the specific message here @angela-tran, anything action-oriented for the user and @indexing can always give feedback via the design process as we evolve the interface.